docs/advanced/input_files/input-main.md

```diff
@@ -395,10 +395,19 @@ These variables are used to control general system parameters.
 
 - **Type**: Integer
 - **Description**: takes value 1, 0 or -1.
-  - if set to 1, symmetry analysis will be performed to determine the type of Bravais lattice and associated symmetry operations. (point groups only)
-  - if set to 0, only time reversal symmetry would be considered in symmetry operations, which implied k point and -k point would be treated as a single k point with twice the weight.
-  - if set to -1, no symmetry will be considered.
-- **Default**: 0
+  - 1: No symmetry will be considered.
+  - 0: Only time reversal symmetry would be considered in symmetry operations, which implied k point and -k point would be treated as a single k point with twice the weight.
+  - 1: Symmetry analysis will be performed to determine the type of Bravais lattice and associated symmetry operations. (point groups, space groups, primitive cells, and irreducible k-points)
+- **Default**: 
+  - 0: if *[calculation](#calculation)=md/nscf/istate/ienvelope/get_S*
+  - 1: else
```

Comment:
What is the default value when gamma_only=1

Reply:
> Symmetry does not dependent on gamma_only, because point/space group analysis and rho/force/stress symmetrization can still be needed when gamma_only=1.

My understanding is that performing symmetry analysis in the program can save the total computational cost. Have you tested the trade-off issue of the computational cost of the symmetry operation itself in large-scale calculations?

Reply:
> My understanding is that performing symmetry analysis in the program can save the total computational cost. Have you tested the trade-off issue of the computational cost of the symmetry operation itself in large-scale calculations?

No I haven't. I only know that symmetrization is slow when the configuration has too many supercells.
OK, I will set a non-forced default value of symmetry to 0 when gamma_only=1.